src/main/java/edu/harvard/iq/dataverse/search/IndexServiceBean.java

```diff
@@ -1253,6 +1268,7 @@ private String addOrUpdateDataset(IndexableDataset indexableDataset, Set<Long> d
                     }
                 }
             }
+            solrInputDocument.addField(SearchFields.EMBARGO_END_DATE, embargoEndDate.toEpochDay());
```

Comment:
I'm getting a null pointer here because none of my files are embargoed(?)

Reply:
Sorry - try that - I switched to using toEpochDay at the end and forgot to check if embargoEndDate was null.